### PR TITLE
books: rules update

### PR DESCRIPTION
**Pull request by kprzerwa**
* fixes broken fields
* tested on 100k data set
* fixes migration additional info object

### CODE REVIEW

File: cds_dojson/marc21/fields/books/base.py [hunk 1]
```diff
@@ -362,16 +377,21 @@ def accelerator_experiments(self, key, value):
...
-    try:
-        clean_val('y', value, str, manual=True)
-    except ManualMigrationRequired as e:
-        e.subfield = 't'
-        raise e
+    sub_y = clean_val('y', value, str)
     url = clean_val('u', value, str, req=True)
-    if 'cds.cern.ch' not in url:
+    eitems = self['_migration']['eitems_proxy']
+    if 'https://cds.cern.ch/auth.py' in url and sub_y == 'ebook':
```
Comment (ntarocco) [inline cds_dojson/marc21/fields/books/base.py:383]: do we have to keep this? Chat IRL?

File: cds_dojson/marc21/fields/books/base.py [hunk 2]
```diff
@@ -443,8 +463,11 @@ def alternative_identifiers(self, key, value):
         else:
             raise UnexpectedValue(subfield='2')
     if key == '035__':
+        if 'CERCER' in sub_a:
```
Comment (ntarocco) [inline cds_dojson/marc21/fields/books/base.py:466]: 😱

**Review by ntarocco — COMMENTED** [review]
In another PR, please do rename `CdsBooks` or `books` or similar to `CDS-ILS`

File: cds_dojson/marc21/fields/books/base.py [hunk 3]
```diff
@@ -443,8 +463,11 @@ def alternative_identifiers(self, key, value):
...
+            raise IgnoreKey('alternative_identifiers')
         sub_9 = clean_val('9', value, str, req=True)
-
+        if 'CERCER' in sub_9:
```
Comment (topless) [inline cds_dojson/marc21/fields/books/base.py:469]: nit: the obvious

**Review by topless — APPROVED** [review]

File: cds_dojson/marc21/fields/books/base.py (hunk 3 above)
Comment (kprzerwa) [reply cds_dojson/marc21/fields/books/base.py:469]: I'm sorry, could you elaborate on the obvious? I don't know what you mean

File: cds_dojson/marc21/fields/books/base.py (hunk 1 above)
Comment (kprzerwa) [reply cds_dojson/marc21/fields/books/base.py:383]: I would leave it like this, since it has to be handled on the ILS side. WDYT?

**Review by ntarocco — APPROVED** [review]

**Comment by ntarocco** [comment]
Just a few comments, discussed by chat, they can go in in the next PR!

**Pull request merged**